### DISCUSSION TITLE
49: take-ownership: not found

### DISCUSSION THREAD

**Discussion by yesBad** [jlesage/docker-handbrake#271] [Q&A]
So this is the issue I'm getting and I'm not sure what the "[take-ownership](https://github.com/jlesage/docker-handbrake/blob/1ccf7d56af2dd3e5fabfedc2569383cf1846b5fb/rootfs/etc/cont-init.d/55-handbrake.sh#LL49C5-L49C42)" command is supposed to do as I've never heard of the command before nor can I find anything with Googling around... It appears to be a Windows thing and I'm quiet confused...

If anyone has ideas let me know..

**Comment by jlesage** [comment]
`take-ownership` is a script part of the container.  What is the issue exactly ?  You may want to share the container's log along with the way you created the container.

**Reply by yesBad** [reply]
> `take-ownership` is a script part of the container. What is the issue exactly ? You may want to share the container's log along with the way you created the container.

<details>
<summary>Logs</summary>
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-app-niceness.sh: executing... 
[cont-init.d] 00-app-niceness.sh: exited 0.
[cont-init.d] 00-app-script.sh: executing... 
[cont-init.d] 00-app-script.sh: exited 0.
[cont-init.d] 00-app-user-map.sh: executing... 
[cont-init.d] 00-app-user-map.sh: exited 0.
[cont-init.d] 00-clean-logmonitor-states.sh: executing... 
[cont-init.d] 00-clean-logmonitor-states.sh: exited 0.
[cont-init.d] 00-clean-tmp-dir.sh: executing... 
[cont-init.d] 00-clean-tmp-dir.sh: exited 0.
[cont-init.d] 00-set-app-deps.sh: executing... 
[cont-init.d] 00-set-app-deps.sh: exited 0.
[cont-init.d] 00-set-home.sh: executing... 
[cont-init.d] 00-set-home.sh: exited 0.
[cont-init.d] 00-take-config-ownership.sh: executing... 
[cont-init.d] 00-take-config-ownership.sh: exited 0.
[cont-init.d] 00-xdg-runtime-dir.sh: executing... 
[cont-init.d] 00-xdg-runtime-dir.sh: exited 0.
[cont-init.d] 10-certs.sh: executing... 
[cont-init.d] 10-certs.sh: exited 0.
[cont-init.d] 10-cjk-font.sh: executing... 
[cont-init.d] 10-cjk-font.sh: exited 0.
[cont-init.d] 10-nginx.sh: executing... 
[cont-init.d] 10-nginx.sh: exited 0.
[cont-init.d] 10-vnc-password.sh: executing... 
[cont-init.d] 10-vnc-password.sh: exited 0.
[cont-init.d] 10-web-index.sh: executing... 
[cont-init.d] 10-web-index.sh: exited 0.
[cont-init.d] 54-check-optical-drive.sh: executing... 
looking for usable optical drives...
no usable optical drive found.
[cont-init.d] 54-check-optical-drive.sh: exited 0.
[cont-init.d] 54-check-qsv.sh: executing... 
Processor: Intel(R) Xeon(R) CPU E5-2690 0 @ 2.90GHz
Intel Quick Sync Video not supported: device directory /dev/dri not exposed to the container.
[cont-init.d] 54-check-qsv.sh: exited 0.
[cont-init.d] 54-check-trash-dir.sh: executing... 
/var/run/s6/etc/cont-init.d/54-check-trash-dir.sh: 9: is-bool-val-true: not found
[cont-init.d] 54-check-trash-dir.sh: exited 0.
[cont-init.d] 55-handbrake.sh: executing... 
core dump file location: |/usr/share/apport/apport -p%p -s%s -c%c -d%d -P%P -u%u -g%g -- %E
core dump file size:  (blocks)
/var/run/s6/etc/cont-init.d/55-handbrake.sh: 49: take-ownership: not found
[cont-init.d] 55-handbrake.sh: exited 127.
[services.d] stopping services
[services.d] stopping s6-fdholderd...
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] syncing disks.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
</details

**Comment by jlesage** [comment]
Looks like you are using an old version of the container...  Did you tried the latest version ?

**Reply by yesBad** [reply]
Pretty sure I tried latest, but I'll try again :)

**Comment by jlesage** [comment]
The log you provided indicate that it's an old version (s6 has been removed like 4 months ago).

**Reply by yesBad** [reply]
Oh lmao, well then I shall try again :)
